pkg/options/options.go

```diff
@@ -30,7 +30,7 @@ const (
 	// ContainerImageRepo is the repo of the default image url
 	ContainerImageRepo = "noobaa-core"
 	// ContainerImageTag is the tag of the default image url
-	ContainerImageTag = "5.9.0-20210722"
+	ContainerImageTag = "5.9.0-20211118"
```

Comment:
@liranmauda why is this image tag in the future? 🤔

Reply:
Being a good developer means using future, unwritten, undeveloped builds 😂 

Anyway, it was supposed to be 20211018, you can change that to the latest...